web/src/javascript/game.js

```diff
@@ -1,18 +1,15 @@
 import {
-  drawCellsOnMap,
+  drawFoodOnMap,
   drawMap,
   drawMiniMap,
+  drawPlayersOnMap,
+  initMap,
 } from "./drawMap";
 
-function extractPointsFromState(gameState) {
-  return [].concat.apply(...gameState.players.map((p) => p.cells));
-}
-
-export function draw(gameState) {
-  drawCellsOnMap(extractPointsFromState(gameState));
-  drawMap();
-  drawMiniMap();
-}
-
-export function update(gameState) {
+export function draw(gameState, canvas) {
+  initMap(canvas);
+  drawFoodOnMap(gameState.food, canvas);
+  drawPlayersOnMap(gameState.players, canvas);
+  drawMap(canvas);
+  drawMiniMap(canvas);
```

Comment:
Oh, yes you are right. But to be honnest I don't know how to. The map needs to be drawn after the food, but before the minimap. Let's do an issue for that.